.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+  "directory": "public/lib"
```

Comment:
If you want the bower components to be installed in `client/lib` this needs to be:

```
{
  "directory": "client/lib"
}
```